packages/s3-store/index.ts

```diff
@@ -177,19 +177,15 @@ export class S3Store extends DataStore {
     return data.ETag as string
   }
 
-  private async getIncompletePart(id: string) {
+  private async getIncompletePart(id: string): Promise<Readable | undefined> {
     try {
       const data = await this.client.getObject({
         Bucket: this.bucket,
         Key: this.partKey(id, true),
       })
-      return data.Body?.transformToByteArray()
+      return data.Body as Readable
     } catch (error) {
-      if (
-        error.Code === 'NoSuchKey' ||
-        error.Code === 'NoSuchUpload' ||
```

Comment:
I think we also had `NoSuchUpload` for S3-compatible storages who return a different code

Reply:
I was looking for any previous discussion on this topic but could not find it. We can keep all types of error, but eventually I'd like to check which are actually required.

`NoSuchUpload` is returned for multipart upload operations where you provide `UploadId`.

Reply:
I also can't find it anymore. So I'm okay with simplifying 👍